soh/src/code/z_player_lib.c

```diff
@@ -1935,6 +1947,20 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
                         Player_DrawHookshotReticle(play, this, RETICLE_MAX);
                     }
                 }
+            } else if (CVarGetInteger(CVAR_ENHANCEMENT("BoomerangReticle"), 0) && (this->heldItemAction == PLAYER_IA_BOOMERANG)) {
+                if (Player_HoldsBoomerang(this)) {
+                    if (LINK_IS_ADULT) {
+                        Matrix_RotateZYX(-31200, -9200, 17000, MTXMODE_APPLY);
```

Comment:
Could you please make this magic values a constexpr floats with names or put them into config section?

Reply:
All set - this should probably also happen for the Bow section that this PR was modeled after, but I'm not adding it to this one.